### PR TITLE
Rendererclient

### PR DESCRIPTION
**Pull request by RussTorres**
This includes a client that allows you to render based on renderparameters, as well as api calls to retrieve and render them.

### CODE REVIEW

File: integration_tests/test_client_integrated.py [hunk 1]
```diff
@@ -361,6 +361,29 @@ def test_processpools_parallelfuncs(
...
+    renderparams = renderapi.image.get_bb_renderparams(
+        teststack, z, x, y, width, height,
+        scale=0.25, render=render)
+    yield renderparams
+
+
+@pytest.fixture(scope='module')
+def renderedimg_renderparams(render, example_renderparameters):
+    # TODO is there a "render from renderparams" api?
+    arr = renderapi.image.get_renderparameters_image(
```
Comment (fcollman) [inline integration_tests/test_client_integrated.py]: I think the api on render side is just called "render" in the sense that rendering from render_parameters was sort of the core of things.  I would argue we might want to call this renderapi.image.render(renderparameters)

Comment (RussTorres) [reply integration_tests/test_client_integrated.py]: Are you refering to the "TODO" note here?  I added that api call but forgot to remove the comment.

**Comment by RussTorres** [comment]
What are you proposing?  Writing up a more formal treatment of renderparameters?  Or restructuring to make it easier for python to do heavy lifting (coordinate mapping, bbox calculations, rendering, etc.)?

The missing piece there is the python mesh and rendering (which are coming soon in separate PRs), but they're still probably pretty experimental to justify a large refactor effort.

I also would like to push this through as I am using the rendererClient to do sparse materialization and dynamically rendered transfers to cloudvolume.

**Comment by fcollman** [comment]
Yeah I see what you are saying, I think we are on the same page on wanting that client side capability at some point but I see we don't really have the pieces in play yet

**Pull request merged**